packages/mobile/src/components/lineup-tile/CollectionTileTrackList.tsx

```diff
@@ -0,0 +1,147 @@
+import { LineupTrack } from 'audius-client/src/common/models/Track'
+import { range } from 'lodash'
+import { Pressable, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import Skeleton from 'app/components/skeleton'
+import { getPlayingUid } from 'app/store/audio/selectors'
+import { flexRowCentered, makeStyles } from 'app/styles'
+import { GestureResponderHandler } from 'app/types/gesture'
+
+// Max number of tracks to display
+const DISPLAY_TRACK_COUNT = 5
+
+type LineupTileTrackListProps = {
+  isLoading?: boolean
+  onPress: GestureResponderHandler
+  trackCount?: number
+  tracks: LineupTrack[]
+}
+
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
+  item: {
+    ...flexRowCentered(),
+    width: '100%',
+    height: spacing(7),
+    fontSize: typography.body,
+    overflow: 'hidden',
+    paddingHorizontal: spacing(2),
+    fontWeight: typography.fontByWeight.medium
+  },
+
+  text: {
+    ...typography.body2,
+    color: palette.neutralLight4,
+    lineHeight: spacing(7)
+  },
+
+  title: {
+    color: palette.neutral,
+    maxWidth: '60%'
```

Comment:
makes sense, is there not a better way for the width stuff?

Reply:
I might be able to pull it off with flex, basically this was to get the truncation working. But it's a little weird because even in the existing app, it truncates the artist name even when there is enough space due to the track name being short. Lemme see if I can improve that